src/Valet.UnitTests/AppTests.cs

```diff
@@ -35,7 +35,8 @@ public async Task CheckForUpdates_NoUpdatesNeeded(string? latestImage, string? c
         var image = "valet-customers/valet-cli";
         var server = "ghcr.io";
 
-        var stringWriter = new StringWriter();
+        using var stringWriter = new StringWriter();
+        var out = Console.Out;
```

Comment:
`out` is a reserved key name. This same could be accomplished across all test methods by using a tear down method. E.g.

```csharp
[TearDown]
public void AfterEachTest()
{
  Console.SetOut(Console.Out);
}
```